internal/db/manager.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"log"
 
-	_ "github.com/lib/pq"
```

Comment:
is this bc we didn't want to connect things before?

Reply:
To be honest, I don't actually know what this syntax means in Go. I'm calling some methods from the `pq` package now, since that appeared to be necessary in order to put an array value in a prepared SQL statement, and when I attempted to call that method, my editor provided an autofix suggestion to import this package under its default name of `pq`.

Reply:
If you don't need to explicitly interact with anything from the package you can alias it as `_`, if you are calling methods there then you can alias it or just remove that `_`